lib/logger-helper.js

```diff
@@ -31,7 +31,7 @@ var auditRequest = function (req, options) {
             auditObject = Object.assign(auditObject, req.additionalAudit);
         }
 
-        options.logger.info(auditObject, 'Inbound Transaction');
+        options.logger.info(auditObject, 'Inbound Transaction - request start');
```

Comment:
wouldn't you like to also add a suffix to `request end`?

Reply:
wouldn't want to break the current used by all string, might have alerts based on it etc..

Reply:
changed the solution, please re-review